include/cantera/oneD/refine.h

```diff
@@ -69,51 +123,132 @@ class Domain1D;
+     * grid is copied directly.
+     *
+     * @param[in] n The number of points in the original grid array `z`.
+     * @param[in] z Pointer to the array of original grid points.
+     * @param[in] nn The maximum number of points that the new grid array `znew` can hold.
+     * @param[out] znew Pointer to the array where the new grid points will be stored.
+     *
+     * @return The function returns 0 upon successful creation of the new grid. Throws an exception if the provided
+     *         output array size is insufficient to hold the new grid.
+     */
     int getNewGrid(int n, const double* z, int nn, double* znew);
-    int nNewPoints() {
-        return static_cast<int>(m_loc.size());
+
+    /**
+     * Returns the number of new grid points that were needed.
+     */
+    int numNewPoints() {
+        return static_cast<int>(m_insertion_points.size());
     }
+
+    /**
+     * Displays the results of the grid refinement analysis.
+     *
+     * This method logs information about where new grid points have been inserted
+     * and the components that required these insertions, if any. If no new points
+     * were needed, it logs that the current grid is sufficient.
+     *
+     * @note This method should be called after analyze() to report the outcomes of
+     * the refinement analysis.
+     */
     void show();
+
+    /**
+     * Returns true if a new grid point is needed to the right of grid index j.
+     *
+     * @param j  Grid point index
+     */
     bool newPointNeeded(size_t j) {
-        return m_loc.find(j) != m_loc.end();
+        return m_insertion_points.find(j) != m_insertion_points.end();
     }
+
+    /**
+     * Returns true if the grid point at index j should be kept.
+     *
+     * @param j Grid point index
+     */
     bool keepPoint(size_t j) {
         return (m_keep[j] != -1);
     }
-    double value(const double* x, size_t i, size_t j);
 
+    /**
+     * Returns the value of the solution component, n, at grid point j.
+     *
+     * @param x Solution vector
+     * @param n Solution Component index
+     * @param j Grid point index
+     */
+    double value(const double* x, size_t n, size_t j);
+
+    //! Returns the maximum ratio of grid spacing at adjacent intervals
     double maxRatio() {
         return m_ratio;
     }
+
+    //! Returns the maximum allowable difference in value between adjacent points
     double maxDelta() {
         return m_slope;
     }
+
+    //! Returns the maximum allowable difference in the derivative between adjacent points
     double maxSlope() {
         return m_curve;
     }
+
+    //! Returns the threshold for removing unnecessary grid points
     double prune() {
         return m_prune;
     }
 
 protected:
     //! Indices of grid points that need new grid points added after them
-    set<size_t> m_loc;
+    set<size_t> m_insertion_points;
+
+    //! Map of grid point indices that should be kept, 1=keep, -1=remove, 0=unset
```

Comment:
```suggestion
    //! Map of grid point indices that should be kept. Values are ...
```
Detailed description should not be part of the first statement, which forms Doxygen's `@brief` description. Also, which integer is what? 

As an aside, using an `enum` as the value instead of these sentinel values would further clarify things.